src/Token/AccessToken.php

```diff
@@ -34,11 +34,11 @@ class AccessToken
      */
     public function __construct(array $options = null)
     {
-        if (! isset($options['access_token'])) {
-            throw new \InvalidArgumentException(
-                'Required option not passed: access_token'.PHP_EOL
-                .print_r($options, true)
-            );
+        if (empty($options['access_token'])) {
+            throw new InvalidArgumentException(sprintf(
+                'Required option not passed: "%s"',
+                'access_token'
```

Comment:
I think `InvalidArgument` makes most sense here.

I understand the readability on the `sprintf()` call, but I think it's unnecessary to use a function call when it's not needed. We can leave it in for now, but we should eventually do some benchmarks and look at call graphs to see if we can identify any bottlenecks and clean them up (I'm not saying `sprintf()` is a bottleneck, but it's an extra function call that can be avoided).